products/bash/release-products.sh

```diff
@@ -568,6 +570,29 @@ function release_tracing {
     helm install ${tracing_chart}/ --name ${tracing_release_name} --namespace ${tracing_namespace} --values tracing-values.yaml --tls
 }
 
+# --- Postgres ---------------------------------------------------------
+
+function install_postgres {
+  echo "Installing PostgreSQL..."
+
+  cat << EOF > postgres.env
+  MEMORY_LIMIT=2Gi
+  NAMESPACE=openshift
+  DATABASE_SERVICE_NAME=postgresql
+  POSTGRESQL_USER=admin
+  POSTGRESQL_USER=password
```

Comment:
This is setting the USER, should be setting the password

Reply:
<img width="705" alt="Screenshot 2020-06-05 at 13 27 05" src="https://user-images.githubusercontent.com/20361559/83876145-440a0e80-a730-11ea-8eeb-1c866b202ea1.png">